test/e2e/properties.test.js

```diff
@@ -8,7 +9,16 @@ describe('vl-properties', async () => {
         return vlPropertiesPage.load();
     });
 
-    it('dummy test omdat anders de browser vensters niet gesloten worden', () => {
-        assert.isTrue(true);
+    it('Als gebruiker kan ik properties zien in een full-size kolom', async () => {
+        const properties = await vlPropertiesPage.getProperties();
+        const propertiesChildren = await properties.getSlotElements();
+
+        assert.lengthOf(propertiesChildren, 4);
+        const fullSizeColumn = await new VlPropertiesColumn(driver, propertiesChildren[propertiesChildren.length - 1]);
+        await assert.eventually.isTrue(fullSizeColumn.isFullSize());
+
+        const propertiesList = await fullSizeColumn.getPropertiesList();
+        const nationaliteitProperty = await propertiesList.getProperty('Nationaliteit');
+        await assert.eventually.equal(nationaliteitProperty.value.getText(), 'Belg');
     });
```

Comment:
evt nog ne getProperty(index) toevoegen en die bijvoegen in de test, of in de test getProperties()[index] doen. alleszins dat gebruik ook aantonen via de API, want er gaat niet altijd misschien "gewoon tekst" in de term zitten om die te kunnen ophalen ...